[2023-03-13 14:57:56,962][DEBUG]: Starting new HTTP connection (1): localhost:8080
[2023-03-13 14:57:56,969][DEBUG]: http://localhost:8080 "GET / HTTP/1.1" 404 None
[2023-03-13 14:57:56,973][DEBUG]: Reproduce HTTP request with:
[2023-03-13 14:57:56,973][DEBUG]: curl -X POST 'http://localhost:8080/evaluateFromRepoWithParams' --data 'programName=../../../../disk/results/chasebench/ontology-256/vadalog-parsimonious-naive/data/q11/run-2/new_program.vada' --data 'params=NonExist=1' --data 'prop=NonExistingProp=0'
[2023-03-13 14:57:56,974][DEBUG]: Starting new HTTP connection (1): localhost:8080
[2023-03-13 15:07:56,715][ERROR]: Interrupted!
